contrib/meson/meson.build

```diff
@@ -11,11 +11,16 @@
 # The intention is that it can be easily moved to the root of the project
 # (together with meson_options.txt) and packaged for wrapdb.
 
-project('lz4', ['c'],
-  license: ['BSD', 'GPLv2'],
-  default_options : ['c_std=c99',
-    'buildtype=release'],
+project(
+  'lz4',
+  ['c'],
+  license: 'BSD AND GPLv2',
```

Comment:
@Cyan4973 could you pick out the spdx identifiers from this list? https://spdx.org/licenses/ that I can put here?

Reply:
The library is licensed as BSD, so it would be `BSD-2-Clause-Patent`,
but the CLI (Command Line Interface) is licensed as GPLv2, which would be `GPL-2.0-or-later`.